src/pages/home/Home.jsx

```diff
@@ -1,7 +1,23 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import "./home.css";
+import { useNavigate } from "react-router-dom";
+import { useFilter } from "../../contexts/filter/FilterContext";
```

Comment:
you can create index.js file in context and export all context in that file. so you have to not write long path here

Reply:
Sure. I will do that